plugins/ccgx/fu-plugin-ccgx.c

```diff
@@ -12,12 +12,17 @@
 #include "fu-ccgx-firmware.h"
 #include "fu-ccgx-hid-device.h"
 #include "fu-ccgx-hpi-device.h"
+#include "fu-ccgx-dmc-device.h"
+#include "fu-ccgx-dmc-firmware.h"
+
 
 void
 fu_plugin_init (FuPlugin *plugin)
 {
 	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
 	fu_plugin_add_firmware_gtype (plugin, "ccgx", FU_TYPE_CCGX_FIRMWARE);
+        fu_plugin_add_firmware_gtype (plugin, "ccgx", FU_TYPE_CCGX_DMC_FIRMWARE);
```

Comment:
tab

Reply:
Fixed on today commit (ccgx: fix code alignment, modify code for review)